### PR TITLE
very small pistol tweaks

### PR DESCRIPTION
**Pull request by rye-rice**
## About The Pull Request

Makes the ringneck 8 rounds
Makes the commander 12 rounds

Adjust the sprites to match

## Why It's Good For The Game

Apogee and I decided this would be better

## Changelog

:cl:
add: Touches up the ringneck and commander magazine sprites
balance: Ringneck is now 8 rounds and commander is 12 rounds.
/:cl:

### CODE REVIEW

**Comment by rye-rice** [comment]
@Apogee-dev

**Review by Apogee-dev — APPROVED** [review]
i'm apogee and i approve of this message

**Comment by Apogee-dev** [comment]
fix the magazine icons though.

**Comment by rye-rice** [comment]
done, no idea wtf is causing the final fail

File: code/modules/cargo/packs/gun.dm [hunk 1]
```diff
@@ -31,7 +31,7 @@
 
 /datum/supply_pack/gun/commanders
 	name = "Commander Pistol Crate"
-	desc = "Contains a modified Candor 'Commander' pistol, produced by Nanotrasen and chambered in 9mm."
+	desc = "Contains a high capacity'Commander' pistol, produced by Nanotrasen and chambered in 9mm."
```
Comment (FalloutFalcon) [inline code/modules/cargo/packs/gun.dm:34]:
```suggestion
	desc = "Contains a high capacity 'Commander' pistol, produced by Nanotrasen and chambered in 9mm."
```

**Review by FalloutFalcon — APPROVED** [review]

**Comment by rye-rice** [comment]
remaking due to check failure, maybe thats causing it?

**Pull request closed without merging**